sonar/modules/validation/api.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+#
+# Swiss Open Access Repository
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Validation API."""
+
+from datetime import datetime
+
+import pytz
+from flask import current_app
+from flask_babelex import _
+from invenio_i18n.ext import current_i18n
+
+from sonar.modules.users.api import UserRecord, current_user_record
+from sonar.modules.utils import send_email
+
+
+class Validation:
+    """Validation API."""
+
+    STATUS_IN_PROGRESS = 'in_progress'
```

Comment:
`STATUS` and `ACTION` can be an enum implemented using a class: i.e. `STATUS.VALIDATED`